### PR TITLE
feat(meta): ability to pass meta for srw

### PR DESCRIPTION
**Pull request by antmat**
Bound with https://github.com/3Hren/cocaine-core/pull/69
Looks like it's done now

### CODE REVIEW

File: node/include/cocaine/api/stream.hpp [hunk 1]
```diff
@@ -14,15 +16,22 @@ class stream_t {
 
     virtual
     stream_t&
-    write(const std::string& chunk) = 0;
+    write(const std::string& chunk, hpack::header_storage_t headers) = 0;
```
Comment (3Hren) [inline node/include/cocaine/api/stream.hpp]: You know, overloads.

File: node/include/cocaine/api/stream.hpp [hunk 2]
```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <cocaine/hpack/header.hpp>
```
Comment (3Hren) [inline node/include/cocaine/api/stream.hpp]: Actually we don't require definitions here, only forwards.

**Comment by 3Hren** [comment]
LGTM, except the last notice.

File: node/include/cocaine/api/stream.hpp (hunk 2 above)
Comment (antmat) [inline node/include/cocaine/api/stream.hpp]: 1) we pass by value, so we do require definition
2) everybody who uses this interface must be aware of hpack::header_storage_t, as all stream_t methods require it.

Comment (3Hren) [inline node/include/cocaine/api/stream.hpp]: 1. No, we don't.
2. IWYU. For example an implementation can provide its own methods, which encapsulate headers, and provides its API without requirement of headers definition.

**Pull request merged**